File: Uri-Online-Judge/Java-Resolutions/URI_1069.java
import java.util.Scanner;
public class URI_1069 { 
	public static void main(String[] args){
		Scanner input = new Scanner (System.in);
		int cont = input.nextInt();
		for(int i = 0;i<cont;++i){
			int cont2 = 0;
			int qnt = 0;
			String diamante = input.next();
			for (int j = 0;j<diamante.length();++j){
				char letra = diamante.charAt(j);
				if (letra == '<'){
					cont2 += 1;
				}else if(letra == '>'){
					cont2 -= 1;
					if(cont2 >= 0){
						qnt += 1;
					}else{
						cont2 = 0;
					}
				}
			}
			System.out.println(qnt);
		}
	}
}

File: Uri-Online-Judge/Python-Resolutions/URI_1069.py
N = int(input())
for i in range (N):
    d = input()
    p = 0
    cont = 0
    for j in d:
        if j == "<":
            p += 1
        elif j == ">":
            if p > 0:
                cont += 1
                p -= 1
            elif p < 0:
                p = 0
    print (cont)